plugins/producer_plugin/include/eosio/producer_plugin/block_timing_util.hpp

```diff
@@ -118,7 +118,7 @@ namespace block_timing_util {
    // Return the *next* block start time according to its block time slot.
    // Returns empty optional if no producers are in the active_schedule.
    // block_num is only used for watermark minimum offset.
-   inline std::optional<fc::time_point> calculate_producer_wake_up_time(uint32_t cpu_effort_us, uint32_t block_num,
+   inline std::optional<fc::time_point> calculate_producer_wake_up_time(fc::microseconds cpu_effort_us, uint32_t block_num,
```

Comment:
maybe we can rename the variable `cpu_effort` now that the type enforces microseconds?